recipes/assimp/5.0.0/conanfile.py

```diff
@@ -0,0 +1,156 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class Assimp(ConanFile):
+    name = "assimp"
+    version = "5.0.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/assimp/assimp"
+    description = "A library to import and export various 3d-model-formats including scene-post-processing to generate missing render data."
+    topics = ("conan", "assimp", "3d")
+    license = "BSD-3-Clause"
+
+    requires = "zlib/1.2.11"
+
+    exports_sources = ["CMakeLists.txt"]
+
+    generators = "cmake"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "double_precision": [True, False],
+        "no_export": [True, False],
+        "internal_irrxml": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "double_precision": False,
+        "no_export": False,
+        "internal_irrxml": False,
+        "fPIC": True,
+    }
+
+    _format_option_map = {
```

Comment:
wow, this is a very big set of options! Could you tell why we should have all of these? As the recipe sets its default value to true, maybe we can just hardcode the definitions in the cmake build helper. Are these options something that users may want to turn off?

Reply:
Assimp is a library with a lot of 3d importers (and somes exporters) independent of each other. On Linux, the complete library in release mode has a size of ~200Mo. (~50-60Mo on Windows). I think that users should be able to choose the parts that are useful for their projects in order to obtain a lighter library. however, I think I'll remove `no_export`, which is redundant (user can just disable each exporter) and `internal_irrxml` because we can use the conan package of irrxml.

Reply:
Understood. Thanks for the clarification. Please go ahead with the changes! 😄